high_card.rb

```diff
@@ -3,3 +3,9 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+deck = []
+player = []
+ranks.each { |rank| put decks << ranks}
```

Comment:
also, we want to avoid using `puts`, but even here, `put` is going to throw an error because there isn't a `put` method.